floris/tools/floris_interface.py

```diff
@@ -32,6 +32,9 @@
 from floris.type_dec import NDArrayFloat
 
 
+# from floris.tools import WindData
```

Comment:
Is this needed?

Reply:
This had been reincluded for the purpose of type hinting (see next comment)

Reply:
Ah got it. It's now not comments and used for type hinting. Thanks